### ISSUE TITLE
landscape

### ISSUE THREAD

**Issue by rajanprabu** [ml-struct-bio/cryodrgn#123]
Hi @zhonge, 

This is not a bug report, but I have few questions. 

1) I have tried the new landscape that worked pretty. Made 1000 volumes and clustered them based on the similarity. Now If I want to convert them in to energy landscape I have to write my own code ? Any pointers towards that would be helpful 

2) I understand that cryodrgn classifies based on refined angles. Questions is how close it has to be.. Can one give angles from Class 3D or it wont be optimal. 

Im sorry for silly questions and thanks for your time and effort.

**Comment by zhonge** [comment]
On your first question, I have yet to release the tool, `cryodrgn analyze_landscape_full`, but it is next on my todos. The tool will map all the particles and produce a jupyter notebook for plotting the energy landscape.

On your second question, do you mean using poses from multiple classes? Are the structures from the different classes aligned? If so, then it might be possible to train using the combined poses, but cryodrgn will separate the two classes if the poses are even slightly misaligned.

**Comment by Guillawme** [comment]
If you look at the histogram of `rlnAngleRot` and `rlnAngleTitle` (for example using [this tool](https://github.com/Guillawme/angdist) I made) from a Class3D job and compare it to the one from a Refine3D job, you will see that Class3D does a very rough job at estimating the poses. So, I would not expect great results by using these poses for a cryoDRGN run, but you could try and see what you get.

**Comment by rajanprabu** [comment]
@zhonge 
Thanks of the quick reply, very much appreciate it. Looking forward to using analyze_landscape_full. Eagerly waiting for it.

@Guillawme 
Great ! This is what I was looking for. Will refine them further to get better estimate of poses. 

Thanks a lot and have a nice day

**Comment by zhonge** [comment]
Sorry about the delay. The landscape analysis tools are available in version 1.1. The documentation for how it works is here: https://www.notion.so/cryodrgn-conformational-landscape-analysis-a5af129288d54d1aa95388bdac48235a

It should be fairly straightforward:
```
# Initial volume space analysis on 1k sampled volumes (~30 min, 1 GPU)
cryodrgn analyze_landscape [workdir] [epoch]

# Full mapping of all particles to the inferred conformational landscape (a few hours, 1 GPU)
cryodrgn analyze_landscape_full [workdir] [epoch]
```

The documentation should explain all the inputs/outputs. Feel free to reach out if you have any questions.

**Comment by rajanprabu** [comment]
Thanks a lot @zhonge. I will use it over the weekend and get back to you if I have questions.
